JavaErickHernandez/Day3/SortedList.java

```diff
@@ -0,0 +1,99 @@
+package bootcamp;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+
+public class SortedList {
+	static String numberOfInputs = "Enter the number of Students: ";
+	static String tryAgainMessage = "Try Again";
+	static String enterName = "Enter name: ";
+	static String enterId = "Enter id: ";
+	static String enterCGPA = "Enter CGPA: ";
+	static String error = "Error!";
+	static ArrayList<Student> students = new ArrayList<Student>();
+
+	public static void numberOfInputs() {
+		System.out.println(numberOfInputs);
+		int studentsCount;
+		do {
+			studentsCount = isNumber();
+		} while (studentsCount <= 0 && studentsCount >= 100000);
+		fillStudent(studentsCount);
+	}
+
+	private static void fillStudent(int pairNumbers) {
+		for (int i = 1; i <= pairNumbers; i++) {
+			System.out.println(enterName);
+			Scanner scanner = new Scanner(System.in);
+			String name;
+			do {
+				name = scanner.next();
+			} while (name.length() < 5 && name.length() > 30);
+			System.out.println(enterId);
+			int id;
+			do {
+				id = isNumber();
+			} while (id <= 0 && id >= 100000);
+			System.out.println(enterCGPA);
+			double cgpa = 0.0;
```

Comment:
cgpa?

Reply:
Thats the name of the variable in the excercise, i will change it